capsules/src/low_level_debug/mod.rs

```diff
@@ -5,7 +5,7 @@ mod fmt;
 
 use core::cell::Cell;
 use kernel::hil::uart::{Transmit, TransmitClient};
-use kernel::{AppId, Grant, ReturnCode};
+use kernel::{AppId, CommandResult, ErrorCode, Grant, ReturnCode};
```

Comment:
Do we still need `ReturnCode`?

Reply:
It is still used by the UART HIL which this capsule implements